android-vjestina-homework-movie-app-data-mock/app/src/main/java/agency/five/codebase/android/movieapp/ui/component/MovieCard.kt

```diff
@@ -0,0 +1,68 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.R
+import agency.five.codebase.android.movieapp.mock.MoviesMock
+import agency.five.codebase.android.movieapp.ui.theme.Shapes
+import agency.five.codebase.android.movieapp.ui.theme.spacing
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Card
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+
+data class MovieCardViewState(
+    val imageUrl: String,
+    val isFavorite: Boolean,
+)
+
+@Composable
+fun MovieCard(
+    modifier: Modifier = Modifier,
+    movieCardViewState: MovieCardViewState,
+    onMovieItemClick: () -> Unit = {}
```

Comment:
You should also pass `onLikeButtonClick` function as parameter and just pass it to `FavoriteButton` composable.